cli/src/Api.ts

```diff
@@ -527,8 +527,8 @@ export default class Api {
     return exists ? await this._api.query.contentDirectory.entityById<Entity>(id) : null
   }
 
-  async dataObjectByContentId(contentId: ContentId): Promise<DataObject | null> {
```

Comment:
I think you in-inadvertently auto-replaced occurrences in client code. The queries will start with a lower case.
Perhaps revert these and leave it for another PR.

Reply:
Thanks for pointing that. I hope it wouldn`t be hard to make them lower case again

Reply:
Done https://github.com/Joystream/joystream/pull/2173/commits/7f1cd93229fb8188bc4055d5579ea9b2c37c7fcc